src/main/java/com/filmsage/filmsage/controllers/JournalController.java

```diff
@@ -14,53 +14,42 @@
 
 @Controller
 public class JournalController {
+    private JournalRepository journalDao;
     private UserRepository userDao;
-    private ReviewRepository reviewDao;
 
-    public JournalController(UserRepository userDao, ReviewRepository reviewDao){
+    public JournalController(UserRepository userDao, JournalRepository journalDao){
         this.userDao = userDao;
-        this.reviewDao = reviewDao;
+        this.journalDao = journalDao;
     }
 
-    @GetMapping("/users")
-    public String showUsers(Model model) {
-        model.addAttribute("allusers", userDao.findAll());
-        return "users/index";
+    @GetMapping("/journals")
+    public String showJournals(Model model) {
+        model.addAttribute("alljournals", journalDao.findAll());
+        return "/journals";
     }
 
-    @GetMapping("/users/create")
+    @GetMapping("/journals/create")
     public String showCreate(Model model) {
-        model.addAttribute("review", new Review());
-        model.addAttribute("reviews", reviewDao.findAll());
-            return "users/create";
+        model.addAttribute("journal", new Journal());
+        model.addAttribute("journals", journalDao.findAll());
+            return "journal/create";
         }
 
-        @GetMapping("/users/{id}")
+        @GetMapping("/journals/{id}")
     public String showReview(@PathVariable long id, Model model){
-        model.addAttribute("singleReview",reviewDao.getById(id));
-        return "reviews/show";
+        model.addAttribute("singleJournal",journalDao.getById(id));
+        return "journals/show";
         }
 
-    @PostMapping("/reviews/{id}/edit")
-    public String submitReviewEdit(@ModelAttribute Review reviewToEdit, @PathVariable long id) {
-        reviewToEdit.setUser(userDao.getById(1L));
-        reviewDao.save(reviewToEdit);
-        return "redirect:/reviews/" + id;
+    @PostMapping("/journal/{id}/edit")
+    public String submitJournalEdit(@ModelAttribute Journal JournalToEdit, @PathVariable long id) {
```

Comment:
all this does is redirect to the individual journal page without actually submitting the edit. there is no get mapping to actually present the edit form to the client. we need a get mapping that actually sends the client to the edit form.

change @ModelAttribute Journal JournalToEdit to just @ModelAttribute Journal journal.
variable names in java are capitalized like "camelCase". Class names are capitalized like "CamelCase". 
this is a small detail but it is very important to be consistent with things like this because inconsistency makes code look ugly and hard to read and recruiters DO NOT LIKE ugly code, even if it works fine.